fbpic/openpmd_diag/inputdeck_diag.py

```diff
@@ -0,0 +1,120 @@
+"""
+
+inputdeck_diag.py
+
+opmd derived diagnostic for saving the input deck
+into each simulation dump.
+
+"""
+
+import sys
+import os
+import numpy as np
+from fbpic.openpmd_diag.generic_diag import OpenPMDDiagnostic
+
+
+class InputDeckDiagnostic(OpenPMDDiagnostic):
```

Comment:
For FBPIC (and Warp/WarpX), we typically use the term "input script", rather than "input deck" (e.g. in documentation)
For this reason could replace "deck" with "script" throughout the file? Also could you rename the current file?
(`git mv inputdeck_diag.py inputscript_diag.py`)

Reply:
Sorted!